packages/block-library/src/image/block.json

```diff
@@ -68,6 +68,9 @@
 			"source": "attribute",
 			"selector": "figure > a",
 			"attribute": "target"
+		},
+		"allowResize": {
+			"type": "boolean"
 		}
```

Comment:
What if an Image is dragged out of a Gallery? You should be able to resize it once it's no longer in the Gallery. It seems like that wouldn't work with this approach.

Perhaps we should instead use the new block context API for this, with the Gallery block providing an `allowResize` context that the Image block consumes.

Reply:
hadn't considered that. Turns out the image alignment breaks things as well, so have added a context object to enable a gallery to pass down a number of different settings